v2/writer.go

```diff
@@ -0,0 +1,86 @@
+package car
+
+import (
+	"bytes"
+	"context"
+	"github.com/ipfs/go-cid"
+	format "github.com/ipfs/go-ipld-format"
+	car_v1 "github.com/ipld/go-car"
+	"github.com/willscott/carbs"
```

Comment:
I imagine we eventually want to just move the carbs code into this git repo?

Reply:
yes definitely. I'll get things moved in the coming PRs.